src/System.Device.Gpio/System/Device/Spi/Devices/Windows10SpiDevice.cs

```diff
@@ -112,7 +123,17 @@ public override void WriteByte(byte value)
         /// </param>
         public override void Write(ReadOnlySpan<byte> buffer)
         {
-            _winDevice.Write(buffer.ToArray());
+            if (_isInverted)
+            {
+                Span<byte> toSend = stackalloc byte[buffer.Length];
+                buffer.CopyTo(toSend);
+                ReverseByte(toSend);
+                _winDevice.Write(toSend.ToArray());
+            }
```

Comment:
actually here it might be better to always allocate regular array rather than stackallocating, reversing and converting to array